.travis.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branches:
+  only:
+    - master
+
+language: php
+
+matrix:
+  include:
+    - php: 5.5
+      env: RUN_DEPLOYMENT_TESTS=true RUN_DEVSERVER_TESTS=true RUN_CS_FIXER=true
```

Comment:
It doesn't make sense to run devserver with php other than 5.5 because it only supports 5.5. For deployment, it doesn't use PHP for deploy, so no need for testing.